.platform/routes.yaml

```diff
@@ -247,6 +247,11 @@ https://{default}/:
       # Sponsored sites application
       "/administration/pricing/sponsored.html": { "to": "/administration/pricing#sponsored-sites", "code": 301 }
 
+      # Overview, tutorials, and best-practices redirects (Aug 2023)
+      "^/bestpractices(.*)":  { "to": "/learn/bestpractices$1", "code": 301, "regexp": true }
```

Comment:
for all of these, it assumes we'll never have a page directory that starts with any of these words. Otherwise, we'll inadvertently catch them 
Example:
We add a new page `/overview-of-pricing/update.html` will be caught and redirected to `learn/overview-of-pricing/update.html` 
If we're worried about this, we could change them to 
```
"^/bestpractices/(.*)"
```